lib/spack/spack/test/cmd/ci.py

```diff
@@ -43,9 +44,129 @@
 pytestmark = [pytest.mark.not_on_windows("does not run on windows"), pytest.mark.maybeslow]
 
 
+# class CIEnvMock:
```

Comment:
?

Reply:
Forgot that got committed, removed, thanks